routes/users.js

```diff
@@ -0,0 +1,27 @@
+const router = require('express').Router();
+const fs = require('fs');
+const path = require('path');
+
+function readUsersJSON() {
+  return JSON.parse(fs.readFileSync(path.join(__dirname, '../data/users.json'), 'utf8'));
+}
+
+router.get('/users', (req, res) => {
+  const users = readUsersJSON();
+  res.send(users);
+});
+
+router.get('/users/:id', (req, res) => {
+  const { id } = req.params;
+  const users = readUsersJSON();
+  const user = users.find((element) => element._id === id);
```

Comment:
Можно лучше: использовать деструктурирующее присваивание при работе с коллекциями.
```suggestion
  const user = users.find(({ _id }) => _id === id);
```